src/editmap.cpp

```diff
@@ -573,119 +506,136 @@ void editmap::update_view( bool update_info )
-                   map_cache.lm[target.x][target.y].to_string() );
-        mvwprintw( w_info, point( 1, off++ ), _( "apparent light: %.5f (%d)" ),
-                   al.apparent_light, apparent_light );
-        std::string extras;
-        if( veh_in >= 0 ) {
-            extras += _( " [vehicle]" );
-        }
-        if( g->m.has_flag( TFLAG_INDOORS, target ) ) {
-            extras += _( " [indoors]" );
-        }
-        if( g->m.has_flag( TFLAG_SUPPORTS_ROOF, target ) ) {
-            extras += _( " [roof]" );
-        }
+        off++; // 3
+    }
+    const auto &map_cache = g->m.get_cache( target.z );
+
+    mvwprintw( w_info, point( 1, off++ ), _( "dist: %d u_see: %d v_in: %d scent: %d" ),
+               rl_dist( g->u.pos(), target ), static_cast<int>( g->u.sees( target ) ),
+               veh_in, g->scent.get( target ) );
+    mvwprintw( w_info, point( 1, off++ ), _( "sight_range: %d, daylight_sight_range: %d," ),
+               g->u.sight_range( g->light_level( g->u.posz() ) ),
+               g->u.sight_range( current_daylight_level( calendar::turn ) ) );
+    mvwprintw( w_info, point( 1, off++ ), _( "transparency: %.5f, visibility: %.5f," ),
+               map_cache.transparency_cache[target.x][target.y],
+               map_cache.seen_cache[target.x][target.y] );
+    map::apparent_light_info al = map::apparent_light_helper( map_cache, target );
+    int apparent_light = static_cast<int>(
+                             g->m.apparent_light_at( target, g->m.get_visibility_variables_cache() ) );
+    mvwprintw( w_info, point( 1, off++ ), _( "outside: %d obstructed: %d" ),
+               static_cast<int>( g->m.is_outside( target ) ),
+               static_cast<int>( al.obstructed ) );
+    mvwprintw( w_info, point( 1, off++ ), _( "light_at: %s" ),
+               map_cache.lm[target.x][target.y].to_string() );
+    mvwprintw( w_info, point( 1, off++ ), _( "apparent light: %.5f (%d)" ),
+               al.apparent_light, apparent_light );
+    std::string extras;
+    if( veh_in >= 0 ) {
+        extras += _( " [vehicle]" );
+    }
+    if( g->m.has_flag( TFLAG_INDOORS, target ) ) {
+        extras += _( " [indoors]" );
+    }
+    if( g->m.has_flag( TFLAG_SUPPORTS_ROOF, target ) ) {
+        extras += _( " [roof]" );
+    }
 
-        mvwprintw( w_info, point( 1, off ), "%s %s", g->m.features( target ).c_str(), extras );
-        // 9
-        off++;
-        for( auto &fld : *cur_field ) {
-            const field_entry &cur = fld.second;
-            mvwprintz( w_info, point( 1, off ), cur.color(),
-                       _( "field: %s L:%d[%s] A:%d" ),
-                       cur.get_field_type().id().c_str(),
-                       cur.get_field_intensity(),
-                       cur.name(),
-                       to_turns<int>( cur.get_field_age() )
-                     );
-            off++; // 10ish
-        }
+    mvwprintw( w_info, point( 1, off ), "%s %s", g->m.features( target ).c_str(), extras );
```

Comment:
```suggestion
    mvwprintw( w_info, point( 1, off ), "%s %s", g->m.features( target ), extras );
```

Reply:
These code were working before so I didn't check them, only changed their indentation because I removed an if clause... Your suggestions are viable though, so I'll apply them.